script/Ciel/Day 13/13_01_01_CIEL13_2.txt

```diff
@@ -56,12 +56,12 @@ I put all my strength into protecting her.
 [sha:d8f4aa9067297face8e64190a64e5036db3b5712]{
 -- Page 79, Offset 33354.
 -- なのに、何一つ<正常|マトモ>には働かなかった。
-And yet, nothing was working as normal.
+And yet, everything is refusing to work as normal.
```

Comment:
And yet, nothing is working like [normal/it should]. (my pref) or
And yet, everything is refusing to work normally.